readme.md

```diff
@@ -378,6 +378,19 @@ alfy.config.get('unicorn');
 //=> '🦄'
 ```
 
+#### userConfig
+
+Type: `object`
+
+If the workflow supports user configuration, it will export a readable [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) with the user workflow configuration.
```

Comment:
Shouldn't it **always** export the `Map`? Only it will be empty if the workflow doesn't support user config. It's also not clear what it means for a workflow to "support user configuration".

Reply:
You are right, it seems that it does export a `Map` always.